lib/modules/manager/terraform/util.ts

```diff
@@ -18,6 +18,12 @@ export function checkIfStringIsPath(path: string): boolean {
   return !!match;
 }
 
+const ociRegex = regEx(/^oci:\/\//);
+export { ociRegex };
+export function checkIfChartIsOCI(chart: string): boolean {
```

Comment:
For that, I'd need to either create a new [Repository](https://github.com/renovatebot/renovate/blob/142bebc1eb2bf2fc6d4b0a4bfe6befb098697500/lib/modules/manager/helmv3/types.ts#L7) with `helmRelease.chart` which would not really be correct since the chart is not a repository or extend [`isOCIRegistry`](https://github.com/renovatebot/renovate/blob/142bebc1eb2bf2fc6d4b0a4bfe6befb098697500/lib/modules/manager/helmv3/utils.ts#L85) to also accept strings and `undefined`. 

Are we fine doing the latter?

Reply:
I would prefer that the existing function is extended to accept both a string or a `Repository` object. @viceice WDYT?

Reply:
I implemented this in https://github.com/renovatebot/renovate/pull/19869/commits/0a7707dd30b88ffe9753a7b3d19f390e74620bde, so you can resolve the discussion if y'all are fine with it 😊